code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -73,10 +73,11 @@
 
 /obj/item/weapon/gun/projectile/automatic/sts35/update_icon(var/ignore_inhands)
 	..()
-	if(istype(ammo_magazine,/obj/item/ammo_magazine/SVD))

```

Comment:
But we still have two types of magazines that fit into the gun, neither of which are the correct icon for 556 mags.